src/urllib3/connectionpool.py

```diff
@@ -1091,7 +1092,22 @@ def _validate_conn(self, conn: BaseHTTPConnection) -> None:
         if conn.is_closed:
             conn.connect()
 
-        if not conn.is_verified:
+        if (
+            conn.proxy
+            and conn.proxy.scheme == "https"
+            and conn.proxy_is_verified is False
+        ):
+            warnings.warn(
+                (
+                    f"Unverified HTTPS request is being made using proxy host '{conn.host}'. "
+                    "Adding certificate verification is strongly advised. See: "
+                    "https://urllib3.readthedocs.io/en/latest/advanced-usage.html"
+                    "#tls-warnings"
```

Comment:
Shouldn't it contain `conn.proxy.host` instead of `conn.host`?

It looks like a docs section at that link needs to mention `InsecureProxyWarning` as it does for `InsecureRequestWarning`.

Reply:
Yes, thanks for the catch. I will update the docs.